robot.py

```diff
@@ -37,12 +37,7 @@ def teleopPeriodic(self) -> None:
         drive_y = -rescale_js(self.gamepad.getLeftX(), 0.1) * Chassis.max_wheel_speed
         drive_z = -rescale_js(self.gamepad.getRightX(), 0.1, exponential=2) * spin_rate
         local_driving = self.gamepad.getBButton()
-
-        if self.gamepad.getYButtonPressed():
```

Comment:
could you restore these, also in the past left/right bumpers have been handy for deploying/retracting intake